packages/atomic/src/pages/index.html

```diff
@@ -369,11 +369,10 @@
             <atomic-no-results></atomic-no-results>
           </atomic-layout-section>
           <atomic-layout-section section="pagination">
-            <atomic-load-more-results></atomic-load-more-results>
-            <!-- Alternative pagination
-              <atomic-pager></atomic-pager>
-              <atomic-results-per-page></atomic-results-per-page>
-              -->
+            <!-- <atomic-load-more-results></atomic-load-more-results>
+            Alternative pagination -->
+            <atomic-pager></atomic-pager>
```

Comment:
🙅 😄  load more result is recommended to be used by UX for search pages

Reply:
ah sorry, I was actually just testing and forgot to change it back 😇